.jenkins/pipelines/Azure/Linux/Jenkinsfile

```diff
@@ -68,8 +68,18 @@ def ACCTest(String label, String compiler, String build_type, List extra_cmake_a
                 checkout scm
                 if (fresh_install) {
                     sh  """
-                        sudo bash scripts/ansible/install-ansible.sh
-                        sudo \$(which ansible-playbook) scripts/ansible/oe-contributors-acc-setup.yml
+                        # Install ansible
+                        for i in 1 2 3 4 5
+                        do
+                            sudo scripts/ansible/install-ansible.sh && break
+                            sleep 15
+                        done
+                        # Run ACC Playbook
+                        for i in 1 2 3 4 5
+                        do
+                            sudo \$(which ansible-playbook) scripts/ansible/oe-contributors-acc-setup.yml && break
+                            sleep 15
```

Comment:
This looks good to me. I'm however curious if there is a way to get ansible itself to keep retrying a task until it succeeds.
The failure seems not specific to OE project and hence I'm curious if ansible itself provides a solution.

Reply:
Ansible does have a mechanism for retries: https://docs.ansible.com/ansible/latest/user_guide/playbooks_loops.html#retrying-a-task-until-a-condition-is-met

Reply:
Ansible does have a mechanism for retries, which we do [implement ](https://github.com/openenclave/openenclave/blob/4181f16725c02f525fd82e6a781146b02fdeeffa/scripts/ansible/roles/linux/intel/tasks/redhat/sgx-packages-install.yml#L9). and then exits when succeeds.

The prior comment can be interpreted 2 ways, one of which is the above and the other is to retry indefinitely. Retry forever? Nope, retry x amount of times, yes.